login.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="ko">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Login</title>
+  <link rel="stylesheet" href="login.css">
+</head>
+<body>
+  <div class="loginpage">
+    <div class="text-para">
+     <img src="../images/logo.png" alt=loginimg class="loginimg">
```

Comment:
❗️ 수정요청
`"판다마켓" 클릭 시 루트 페이지("/")로 이동합니다.` 요구사항도 그렇고 일반적인 유저들은 로고를 누르면 home으로 가는 것을 기대합니다~ 이를 위해 a 태그를 추가해주세요!

+이미지의 alt 속성을 기입하실때는 ""를 추가해주셔야 합니다~
또한 alt 속성은 alternative의 약어로 해당 이미지가 여러 이유로 다운로드에 실패했거나 읽히지 못할때 사용하는 대체 텍스트입니다.
따라서 해당 이미지를 설명할 수 있는 문자를 적어주시면 됩니다! 지금의 경우 "판다마켓"이 적절할 것 같습니다~

```suggestion
     <a href="/"><img src="../images/logo.png" alt="판다마켓" class="loginimg"></a>
```